xmtp/Cargo.toml

```diff
@@ -36,8 +38,12 @@ ethers-core = "2.0.4"
 prost = { version = "0.11", features = ["prost-derive"] }
 futures = "0.3.28"
 base64 = "0.21.1"
-tokio = "1.28.1"
+tokio = { version = "1.32.0", features = ["full"] }
```

Comment:
One of the other deps seems to have upgraded us to a version of Tokio that needs a feature to include the `tokio::test` macro, so I added that.